blog/docs/steering/2021-end-user-election.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2021 Steering Committee End User Seat Election
+date: 2021-12-17
+---
+
+Hello, Knative community,
+
+We want to kick off the Knative Steering Committee (SC) End User election, and we are ready to accept nominations from End Users!
```

Comment:
I'm not sure how it's usually done with other posts, but typically, I think you would only use `(SC)` if you subsequently then use only `SC` to refer to the Steering Committee in later text. Since you spell out `Steering Committee (SC)` everywhere else, I think you can eliminate `(SC)` altogether from this line and all of the others.
```suggestion
We want to kick off the Knative Steering Committee End User election, and we are ready to accept nominations from End Users!
```